lighthouse-core/audits/dobetterweb/password-inputs-can-be-pasted-into.js

```diff
@@ -45,7 +45,12 @@ class PasswordInputsCanBePastedIntoAudit extends Audit {
     const items = [];
     passwordInputsWithPreventedPaste.forEach(input => {
       items.push({
-        node: {type: 'node', snippet: input.snippet},
+        /** @type {LH.Audit.Details.NodeValue} */
```

Comment:
I don't think this type comment will work: you must wrap the object value in parentheses, and put the comment just before it (look around for similar usages)

you can confirm it may not work by change 'node' to some random gibberish and note that there may not be a type error on the object